test/worldwide/deprecated_time_zone_mapper_test.rb

```diff
@@ -26,5 +26,432 @@ class TimeZoneTest < ActiveSupport::TestCase
+        ["Africa/Bamako", "Africa/Abidjan"],
+        ["Africa/Bangui", "Africa/Lagos"],
+        ["Africa/Banjul", "Africa/Abidjan"],
+        ["Africa/Blantyre", "Africa/Maputo"],
+        ["Africa/Brazzaville", "Africa/Lagos"],
+        ["Africa/Bujumbura", "Africa/Maputo"],
+        ["Africa/Conakry", "Africa/Abidjan"],
+        ["Africa/Dakar", "Africa/Abidjan"],
+        ["Africa/Dar_es_Salaam", "Africa/Nairobi"],
+        ["Africa/Djibouti", "Africa/Nairobi"],
+        ["Africa/Douala", "Africa/Lagos"],
+        ["Africa/Freetown", "Africa/Abidjan"],
+        ["Africa/Gaborone", "Africa/Maputo"],
+        ["Africa/Kampala", "Africa/Nairobi"],
+        ["Africa/Kigali", "Africa/Maputo"],
+        ["Africa/Kinshasa", "Africa/Lagos"],
+        ["Africa/Libreville", "Africa/Lagos"],
+        ["Africa/Lome", "Africa/Abidjan"],
+        ["Africa/Luanda", "Africa/Lagos"],
+        ["Africa/Lubumbashi", "Africa/Maputo"],
+        ["Africa/Lusaka", "Africa/Maputo"],
+        ["Africa/Malabo", "Africa/Lagos"],
+        ["Africa/Maseru", "Africa/Johannesburg"],
+        ["Africa/Mbabane", "Africa/Johannesburg"],
+        ["Africa/Mogadishu", "Africa/Nairobi"],
+        ["Africa/Niamey", "Africa/Lagos"],
+        ["Africa/Nouakchott", "Africa/Abidjan"],
+        ["Africa/Ouagadougou", "Africa/Abidjan"],
+        ["Africa/Porto-Novo", "Africa/Lagos"],
+        ["Africa/Timbuktu", "Africa/Abidjan"],
+        ["America/Anguilla", "America/Puerto_Rico"],
+        ["America/Antigua", "America/Puerto_Rico"],
+        ["America/Argentina/ComodRivadavia", "America/Argentina/Catamarca"],
+        ["America/Aruba", "America/Puerto_Rico"],
+        ["America/Atikokan", "America/Panama"],
+        ["America/Atka", "America/Adak"],
+        ["America/Blanc-Sablon", "America/Puerto_Rico"],
+        ["America/Buenos_Aires", "America/Argentina/Buenos_Aires"],
+        ["America/Catamarca", "America/Argentina/Catamarca"],
+        ["America/Cayman", "America/Panama"],
+        ["America/Coral_Harbour", "America/Panama"],
+        ["America/Cordoba", "America/Argentina/Cordoba"],
+        ["America/Creston", "America/Phoenix"],
+        ["America/Curacao", "America/Puerto_Rico"],
+        ["America/Dominica", "America/Puerto_Rico"],
+        ["America/Ensenada", "America/Tijuana"],
+        ["America/Fort_Wayne", "America/Indiana/Indianapolis"],
+        ["America/Grenada", "America/Puerto_Rico"],
+        ["America/Guadeloupe", "America/Puerto_Rico"],
+        ["America/Indianapolis", "America/Indiana/Indianapolis"],
+        ["America/Jujuy", "America/Argentina/Jujuy"],
+        ["America/Knox_IN", "America/Indiana/Knox"],
+        ["America/Kralendijk", "America/Puerto_Rico"],
+        ["America/Louisville", "America/Kentucky/Louisville"],
+        ["America/Lower_Princes", "America/Puerto_Rico"],
+        ["America/Marigot", "America/Puerto_Rico"],
+        ["America/Mendoza", "America/Argentina/Mendoza"],
+        ["America/Montreal", "America/Toronto"],
```

Comment:
This particular test line does not give much value, as America/Toronto is not supported by Rails (at least, according to the original GH issue that prompted this work).